otj-kafka-tests/src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -124,9 +126,17 @@ public EmbeddedKafkaBroker start()
     private void maybeCreateConsumerOffsets() throws InterruptedException {
         final String consumerOffsets = "__consumer_offsets";
         retry("create consumer offsets", () -> {
-            Map<String, TopicDescription> description = admin.describeTopics(Collections.singleton(consumerOffsets)).all().get(10, TimeUnit.SECONDS);
-            if (description.isEmpty()) {
-                createTopic(consumerOffsets);
+            try {
+                Map<String, TopicDescription> description = admin.describeTopics(Collections.singleton(consumerOffsets)).all().get(10, TimeUnit.SECONDS);
```

Comment:
Interesting so auto create topics is off now? @rushalias ?

Reply:
I don't think in the past the topics were auto created, the check `admin.describeTopics(...)` in the past was used to create if they were not created since it immediately checked for `description.isEmpty()`.

Now the api has changed when you make same call, you get missing UnknownTopicOrPartitionException.